### ISSUE TITLE
Canvas to second screen

### ISSUE THREAD

**Issue by MitPitt** [astrofox-io/astrofox#23]
The program obviously works really well in real time, not just for rendering videos. So is there a way to show the working canvas in full screen to a second monitor? What I mean is that there's the usual interface on your first monitor, and on the second monitor there's only the canvas that takes up the whole screen. This would be extremely useful for live sets and other live activities. 

I think that's somewhat easy to implement.
Great project by the way, thanks.

**Comment by mikecao** [comment]
Yes, that is on the roadmap. To be able to use the app for a VJ setup including reacting to live audio.

**Comment by mikecao** [comment]
Still relevant in 2.0 — there is a single window and no canvas-only output window. With live mode and MIDI now in 2.0 this makes more sense. Keeping open.

**Comment by mikecao** [comment]
This is covered by the PIP feature.
